src/Api/Customers.php

```diff
@@ -117,11 +117,11 @@ public function show(int $id): Response
      * https://magento.redoc.ly/2.4.3-admin/tag/customerscustomerId#operation/customerCustomerRepositoryV1SavePut
      *
      * @param int $id
-     * @param string $params
+     * @param array $params
      * @return Response
      * @throws Exception
      */
-    public function edit(int $id, string $params): Response
+    public function edit(int $id, array $params): Response
```

Comment:
@developh 
I think you're right that `$params` should be an array rather than a string.
It's technically a breaking change, but it's broken in the original code, if you ask me.

@Medalink @bhongid 
What do y'all think?

Reply:
If we stick to the plan to not introduce breaking changes, then a/the solution would be to add a new function that does the correct thing with the correct parameters.